packages/connection-form/src/components/connection-form.tsx

```diff
@@ -461,9 +497,36 @@ function ConnectionForm({
 
         <div className={formContentContainerStyles}>
           <div className={formContentStyles}>
+            {disableEditingConnectedConnection && onDisconnectClicked && (
+              <Banner
+                data-testid="disabled-connected-connection-banner"
+                className={disabledConnectedConnectionBannerStyles}
+              >
+                <div className={disabledConnectedConnectionContentStyles}>
+                  <div>
+                    While connected, you may only personalize your
```

Comment:
This isn't really what "personalize" means, unless we consider connections to be people

Reply:
Fits the dictionary definition.